js/Source/Slider.Modify/Slider.Modify.js

```diff
@@ -73,6 +75,12 @@ Slider.Modify = new Class({
     this.parent(element, knob, options);
     this.buildTargets();
     this.updateSlideFill();
+    if (this.options.roundAfterSnap){
```

Comment:
I'd consider that user (e.g. developer) error. Oddly, the code would be fine, as there's a bottom limit. So if you put a negative number there it would always snap to your lower limit. I will, however, change the line in the behavior to check for > 0 rather than not-zero.